pkg/bus/meseeks.go

```diff
@@ -699,10 +718,29 @@ func (b *Bus) openSeeks(ctx context.Context, receiverID string, tourneyID string
 	if len(sgs) == 0 {
 		return nil, nil
 	}
+
+	receiver, err := b.userStore.GetByUUID(ctx, receiverID)
+	if err != nil {
+		return nil, err
+	}
+
 	log.Debug().Str("receiver", receiverID).Interface("open-matches", sgs).Msg("open-matches")
 	pbobj := &pb.SeekRequests{Requests: []*pb.SeekRequest{}}
 	for _, sg := range sgs {
-		pbobj.Requests = append(pbobj.Requests, sg.SeekRequest)
+		seeker, err := b.userStore.GetByUUID(ctx, sg.SeekRequest.User.UserId)
```

Comment:
is there a way to get multiple (and possibly combine this with the receiver Get above)? otherwise each is possibly a separate roundtrip to db

Reply:
or maybe have ListOpenSeeks do an antijoin so that in the first place it doesn't return games that would be filtered off and we don't even have to load the full user info of everyone currently playing?